scripts/setDefaultIndex.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+
+XHEAD_PARAMS="-i"
+KIBANA_PREFIX="localhost:9200/.kibana"
+NOT_FOUND="not found"
+INDEX_PATTERN="{\"title\": \"network_*\", \"timeFieldName\": \"@TimeUpdated\", \"customFormats\": {}}"
+DEFAULT_INDEX="{\"doc\":{\"doc\":{\"defaultIndex\":\"network_*\"},\"defaultIndex\":\"network_*\"}}"
+JSON_FLAG="-d"
+
+# Create the index pattern and specify TimeUpdated as the default time parameter
+curl -XPOST localhost:9200/.kibana/index-pattern/network_*/?op_type=create -d "$INDEX_PATTERN"
```

Comment:
shouldn't that be inside the conditional? no need to create the index pattern if it's already set